static/styles/zulip.css

```diff
@@ -86,7 +86,7 @@ pre {
     font-family: "Source Sans 3", sans-serif !important;
 
     /* Affects all tippy tooltips not using any theme. */
-    .tippy-box:not[data-theme] {
+    .tippy-box:not([data-theme]) {
```

Comment:
`.tippy-box:not[data-theme]` is an invalid selector that fails to parse, and all rules containing it are discarded by the browser. This may just be unused, in which case we should delete the entire block.

Reply:
Sorry, actually it doesn't work without brackets, it was hard to notice the change in light mode. We do want the darker color here as was discussed in czo.

Reply:
(Linking said [CZO discussion](https://chat.zulip.org/#narrow/stream/101-design/topic/dark.20mode.20tooltip.20background.20contrast), since it took a while to figure out what discussion you were talking about.)